plugin/src/main/java/io/jenkins/plugins/casc/impl/secrets/VaultSecretSource.java

```diff
@@ -183,10 +183,11 @@ public void setSecrets(Map<String, String> secrets) {
     }
 
     private Optional<String[]> getCommaSeparatedVariables(String key, Properties prop) {
-        if (key.equals(CASC_VAULT_PATH))
+        Optional<String[]> strings = getVariable(key, prop).map(str -> str.split(","));
+        if (key.equals(CASC_VAULT_PATH) && strings.isPresent())
             LOGGER.log(Level.WARNING, "[Deprecation Warning] CASC_VAULT_PATH will be deprecated. " +
```

Comment:
nit pick the tense is wrong in the message if you want to fix now?
```suggestion
            LOGGER.log(Level.WARNING, "[Deprecation Warning] CASC_VAULT_PATH is deprecated. " +
```